pages/index.tsx

```diff
@@ -21,555 +24,282 @@ const LandingPage: FunctionComponent = () => {
       <Head>
-        <title>Getting started with Avo</title>
+        <title>Get Started With Avo</title>
       </Head>
       <div>
         <h1 className={[styles.label, styles.heading1].join(' ')}>
-          Getting started with Avo
+          Get Started With Avo
         </h1>
-        <p className={styles.text}>
-          Avo is where your product analytics starts. Avo is the proactive data
-          planning and governance platform for product analytics. With Avo,
-          teams like Patreon implement 10x faster with autogenerated type safe
-          code and unit tests, sending data exactly as planned.
-        </p>
 
         <p className={styles.text}>
-          Avo will give you and your team:{' '}
-          <b>
-            Better tracking plan management. Tracking validation. Better
-            analytics release workflow
-          </b>
-          . You can adopt all three, or stop at 1 (no engineering effort) or 2
-          (little to no engineering effort).
+          Below are the key areas to consider when getting started with Avo. You
+          can start them in any order or in parallel, and skip those you don't
+          think apply to you.
         </p>
 
-        <div className={styles.tocInline}>{tocContent}</div>
+        {headings != null && headings.length !== 0 && (
+          <div className={styles.inlineToc}>
+            <Toc headings={headings} />
+          </div>
+        )}
 
-        <h2 id="stage1" className={[styles.label, styles.heading2].join(' ')}>
-          Step 1. Tracking plan management – no engineering required
+        <h2 id="workflow" className={[styles.label, styles.heading2].join(' ')}>
+          Try the Avo workflow for your next analytics release
         </h2>
         <p className={styles.text}>
-          Use Avo instead of your spreadsheet to manage and discuss your
-          tracking plan changes. Avo is designed for analytics, so it&apos;s way
-          better than the spreadsheet for managing your tracking plan.
+          If you have an analytics release coming up in the next few days, we
+          highly recommend you try out the Avo Workflow for that release.
         </p>
-
         <p className={styles.text}>
-          This adoption step does not require any developer input, so you can
-          adopt it in the data design team without getting a developer buy-in.
+          The Avo workflow covers all the steps required to plan, review,
+          request implementation, implement, validate and publish your analytics
+          release.
         </p>
-        <h3 className={[styles.label, styles.heading3].join(' ')}>
-          Creating tracking plan in Avo
-        </h3>
+        <p className={styles.callout}>
+          <span>💡</span>
+          When trying the Avo Workflow for the first time, don't choose a
+          product release that's about to take the next 3 months to work on.
+          Choose one that has a handful of new or updated events.
```

Comment:
Should we frame this in a more positive way? Something in this directly perhaps:
```suggestion
          When trying the Avo Workflow for the first time, choose a
          product release that has a handful of new or updated events, rather 
          than something that takes the next 3 montes to work on.
```